src/analysis/convex/specific_functions.lean

```diff
@@ -82,6 +82,30 @@ begin
     nat.sub_pos_of_lt hn) (nat.cast_pos.2 $ zero_lt_two.trans_le hn),
 end
 
+/-- Specific case of Jensen's inequality for sums of powers -/
+lemma real.pow_sum_div_card_le_sum_pow {α : Type*} (s : finset α) (f : α → ℝ) (hf : ∀ a, 0 ≤ f a)
```

Comment:
could you make sure that `hf` only assumes the assumption for a in s, and turn s and f into implicit arguments?